Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "*"
+ipython = "*"
```

Comment:
Since Ipython is NOT required for production deployment, please move it to development dependency